src/guides/all-about-inserts.md

```diff
@@ -260,6 +260,17 @@ On backends that support the `DEFAULT` keyword (all backends except SQLite),
 the data will be inserted in a single query.
 On SQLite, one query will be performed per row.
 
+<aside class = "aside aside--note">
+<header class = "aside__header">A note on SQLite batch inserts:</header>
+
+::: aside__text
+It is possible to perform batch inserts with a SQLite backend. For an `Insertable` struct,
```

Comment:
I would like to formulate this the other way around. Yes this feature has implications on batch inserts on SQLite, but that's not the core thing that might be important here.

Maybe something like that is a better fit?

```suggestion

You can use the `#[diesel(treat_none_as_default_value = false)]` attribute on a struct to disable using `DEFAULT` in place of `None` values and rather use `NULL` values for such fields. This enables for example real batch inserts with the SQLite backend.
```